### ISSUE TITLE
Improvement: Add fonts parsing to the custom fonts filearea

### ISSUE THREAD

**Issue by abias** [moodle-an-hochschulen/moodle-theme_boost_union#117]
This is a follow-up issue to #38 after #116 has been integrated.

The custom fonts filearea does not provide any real additional value beyond the additional resources filearea yet.
This is because the fonts are not parsed yet and the admin still has to compose the CSS snippets manually.

I had a quick look at https://github.com/dompdf/php-font-lib which looked promising at the beginning, but does not provide WOFF2 parsing (which is a must-have from my point of view) yet.

It would be awesome if the theme could automatically parse the uploaded fonts and automatically add the fonts to the CSS for direct usage.
